Resources/ServerInfo/Guidebook/Medical/Medical Doctor.xml

```diff
@@ -0,0 +1,129 @@
+
+2. Is the body rotting? If it is, sadly, they'll have to be cloned. A rotting body can't be brought back with a defibrillator.
+
+3. If they have a soul, and aren't rotten, it's possible to revive them. Not counting airloss, Defibrillator's can be used to revive patients under 200 total damage. Chemicals don't metabolise in the dead, but brute packs and ointment can still patch them up. Use them to bring their numbers down enough for revival.
+
+<Box>
+<GuideEntityEmbed Entity="Brutepack"/>
+<GuideEntityEmbed Entity="Defibrillator"/>
+<GuideEntityEmbed Entity="Ointment"/>
+</Box>
+
+## Cloning
+If all else fails, patients will need to be cloned. This should only be used as a last resort, if recovery is impossible. This is most often the case when the body has started to rot, or if they have excessive amounts of damage.
+
+Cloning itself is relatively simple. (Assuming the network is set.) Just drag a body into the scanner, check the cloning console, and clone! If they've taken cellular damage, or have rotted for too long, the results can be... less than pleasant.
+
+As a note: Cloning isn't always available, and it would be best not to rely on it. Science might be able to set up cloning, if it's unavailable.
+<Box>
+<GuideEntityEmbed Entity="ComputerCloningConsole" Caption="cloning console"/>
+<GuideEntityEmbed Entity="MedicalScanner"/>
+<GuideEntityEmbed Entity="CloningPod"/>
+<GuideEntityEmbed Entity="NetworkConfigurator"/>
+</Box>
+## Biomass
+Creating a clone requires biomass, which, gruesomely, involves recycling organic material, be it crew members without a soul, station pets, or hostile creatures that have been 'taken care of'.
+
+Dragging a body onto the biomass reclaimer will mulch it for re-use. If it's a crew member, it's common practice to strip the body and return departmental gear before doing such.
+<Box>
+<GuideEntityEmbed Entity="BiomassReclaimer"/>
+<GuideEntityEmbed Entity="MaterialBiomass"/>
+</Box>
+## Cryogenics
+Cryogenics can be a bit daunting to understand, but hopefully, quick run through of this diagram, you'll learn how to work it! So, let's go over what each part does:
+
+- air in: Air comes in here through the distro or an air tank, giving those inside something to breathe, and the freezer something to chill.
+- gas pump: Limits the network pressure, so those in the tube aren't crushed.
+- freezer: Chills the air, and by extention, patients in the tube, so Cryoxadone can activate.
+- gas filters: Removes waste gas (Carbon Dioxide and Nitrous Oxide) from the loop.
+- air out: Waste gasses go here, either into a storage tank, or into the waste network.
+<Box>
+<GuideEntityEmbed Entity="GasPipeBend" Rotation="90"/>
+<GuideEntityEmbed Entity="GasPipeStraight" Rotation="90"/>
+<GuideEntityEmbed Entity="GasPipeStraight" Rotation="90"/>
+<GuideEntityEmbed Entity="GasPipeBend" Rotation="0"/>
+<GuideEntityEmbed Entity="CloningPod" Caption="cryo pod" Rotation="0"/>
```

Comment:
Uhhh... is this intended to be correct? There's a dedicated entity prototype for cryo pods, `CryoPod`.

Reply:
why is cryo pod even bundled in with cloning its unrelated